python-package/lightgbm/basic.py

```diff
@@ -21,13 +21,21 @@
 from .libpath import find_lib_path
 
 
+def _log_callback(msg):
+    """Redirect logs from native library into Python console"""
+    print("{0:s}".format(decode_string(msg)), end='')
```

Comment:
it seems python 2 don't support `end=''`. @StrikerRUS @henry0312  any better solution? or we drop the support of python 2?

Reply:
Just add `print_function` to the 3rd line

```
from __future__ import absolute_import, print_function
```